reference-architecture/gce-cli/ansible/playbooks/roles/openshift-ansible-installer/tasks/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: download openshift ansible installer from github
+  git:
+    repo: https://github.com/openshift/openshift-ansible.git
+    dest: '{{ playbook_dir }}/../../../../../openshift-ansible'
```

Comment:
Maybe we can check the installation type. If it is enterprise and RHEL involved, then install the RPMs, otherwise (origin, Fedora, CentOS), go with the git repo.
My 2 cents.